commons/src/lib.rs

```diff
@@ -32,7 +32,8 @@ use url::form_urlencoded;
 lazy_static! {
     /// list of cincinnati versions
     pub static ref CINCINNATI_VERSION: HashMap<&'static str, i32> =
-        [("application/vnd.redhat.cincinnati.v1+json", 1)]
+        [("application/vnd.redhat.cincinnati.v1+json", 1),
+         ("application/vnd.redhat.cincinnati.v2+json", 2)]
```

Comment:
I'm not clear on why we need a v2 for this, since adding a new `conditionalEdges` property is a backwards-compatible change.  Can we keep the conditional-edge handling in v1?

Reply:
I'll mark this thread resolved, and we can sort this out in [the main comment stream][1].

[1]: https://github.com/openshift/cincinnati/pull/574#issuecomment-984053375